dv/verilator/pcount/cpp/ibex_pcounts.h

```diff
@@ -0,0 +1,41 @@
+#ifndef IBEX_PCOUNTS_H_
+#define IBEX_PCOUNTS_H_
+
+#include <cstdint>
+#include <string>
+
+const unsigned int ibex_pcount_num = 11;
```

Comment:
This is a very C way of doing this. It's potentially better to use a std::vector, so when someone adds a new counter, and has to update the list of performance counter names, they won't get issues by forgetting to update this variable?